packages/p2p/src/components/recommend-user/recommend-user.jsx

```diff
@@ -5,18 +5,17 @@ import { Localize } from 'Components/i18next';
 
 const RecommendUser = ({
     is_buy_order_for_user,
-    is_user_rated_previously,
+    is_user_recommended_previously,
     onClickClearRecommendation,
     onClickNotRecommended,
     onClickRecommended,
-    previous_recommendation,
 }) => {
     const [is_no_selected, setIsNoSelected] = React.useState(false);
     const [is_yes_selected, setIsYesSelected] = React.useState(false);
 
     React.useEffect(() => {
-        if (is_user_rated_previously === 0 || is_user_rated_previously === 1) {
-            if (previous_recommendation) {
+        if (is_user_recommended_previously !== null) {
+            if (is_user_recommended_previously) {
```

Comment:
```suggestion
        if (is_user_recommended_previously !== null) {
            if (is_user_recommended_previously) {
```
This is an odd check 🤔  isnt just one falsy check enough for this case? 
```
if (is_user_recommended_previously) {
```

Reply:
I have to check for null as the value `is_recommended` can return null, which is if the user did not select any of the recommend buttons (yes or no), or it returns 1 for yes and 0 for no. If i do it this way, it will select the No button, if the user doesn't select any of the buttons